pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -456,6 +487,16 @@ func (gce *GCECloud) ProviderName() string {
 	return ProviderName
 }
 
+// ProjectID returns the ProjectID corresponding to the project this cloud is in.
+func (g *GCECloud) ProjectID() string {
+	return g.projectID
+}
+
+// NetworkProjectID returns the ProjectID corresponding to the project this cluster's network is in.
+func (g *GCECloud) NetworkProjectID() string {
+	return g.projectID
```

Comment:
Nice catch, thanks.

Yes, `projectNumber` and `networkProjectNumber` aren't used anywhere, but the idea was to make it clear to future authors that there is a difference between `number` and `ID`. I can take them out if you prefer it.